### DISCUSSION TITLE
What do you mean by fit_t_, fit_tau_, fit_tau, and fit_t_ in the parameters of the inference ?

### DISCUSSION THREAD

**Discussion by yujinmath** [theislab/scvelo#415] [Q&A] [answered]
Hi,everyone!
There is a problem that bothers me recently. What do you mean by fit_t_, fit_tau_, fit_tau, and fit_t_ in the parameters of the inference output of the dynamics model?
Thanks in advance for your help!

**Comment by VolkerBergen** [comment, marked as answer]
That's bit cryptic indeed - will update the docs. It's only being used internally, thus a full description if yet missing. Hope this helps for now:

- `fit_t` corresponds to the gene-wise latent times. 
- `fit_t_` are the swicthing timepoints.
- `fit_tau_` are the timepoints of assigning observatins to the off-state (necessary to generate likelihoods after the fitting)
- `fit_tau` same as above, but for the on-state.

**Reply by yujinmath** [reply]
Thanks for your reply. by the way, based on the above fitting timepoints, can we infer the time of the gene from on-state to off-state  and then infer the transcriptional burst frequency of genes?

**Reply by VolkerBergen** [reply]
Timepoints are inferred and can be further leveraged. Transcription rate is being modeled as constant within the window from on- to off-state, so that needs to be extended when modeling transcriptional bursting kinetics.

**Reply by anorak94** [reply]
Hey i was going through the results of the recover dynamics and in the adata object.  `fit_tau_` and `fit_tau` and `fit_t` have the shape `n_cells x n_genes` while `fit_t_` has shape `n_genes`. Since `fit_t_` are the switching time points it makes sense that we have one switching time point per gene and similarly `fit_t` corresponds to the latent time for each cell. But what do `fit_tau_` and `fit_tau` mean. For a cell if its current latent time is greater than `fit_tau_` it is assigned to the off state and this lets us write the log-likelihood to be minimized and the same for `fit_tau` but then how do we estimate these two quantities on a per gene per cell basis. Is it by using the analytical equations. For instance we know alpha_induction_ss and alpha_repression_ss = 0. So we just compute $\tau$ from the analytical equation assuming $t_0 = 0$
